cocotb/__init__.py

```diff
@@ -123,12 +125,8 @@ def _initialise_testbench(root_name):
     if exec_path is None:
         exec_path = 'Unknown'
 
-    version = os.getenv('VERSION')
-    if version is None:
-        log.info("Unable to determine Cocotb version from %s" % exec_path)
-    else:
-        log.info("Running tests with Cocotb v%s from %s" %
-                 (version, exec_path))
+    log.info("Running tests with Cocotb v%s from %s" %
```

Comment:
We (well, I, through my documentation PRs) normalized the casing to an all-lowercase "cocotb",  this is one of the few remaining outliers - mind changing it here?